index.php

```diff
@@ -42,10 +81,12 @@
 
                 <nav class="main-navigation">
                     <ul class="main-navigation__list">
+                        <?php foreach ($projects as $key): ?>
                         <li class="main-navigation__list-item">
-                            <a class="main-navigation__list-item-link" href="#">Название проекта</a>
+                            <a class="main-navigation__list-item-link" href="#"><?= $key; ?></a>
```

Comment:
Здесь поменяется соответсвенно
```suggestion
                            <a class="main-navigation__list-item-link" href="#"><?= $project; ?></a>
```